tools/devops/automation/templates/build/build-pkgs.yml

```diff
@@ -77,11 +77,11 @@ steps:
       name: workload_file
       displayName: 'Generate "WorkloadRollback.json"'
 
-    - task: PublishPipelineArtifact@1
+    - task: 1ES.PublishPipelineArtifact@1
       displayName: 'Publish WorkloadRollback.json'
       inputs:
-        targetPath: $(Build.SourcesDirectory)/WorkloadRollback.json
-        artifactName: '${{ parameters.uploadPrefix }}WorkloadRollback'
+        path: $(Build.SourcesDirectory)/WorkloadRollback.json
+        artifact: '${{ parameters.uploadPrefix }}WorkloadRollback'
```

Comment:
They could have reused the same names :/ Would have made our diff much smaller.

Reply:
Trust me, I have rant about this.

Reply:
They are the same if you use the `templateContext.outputs`, maybe it was intentional to make you move to that format? evil geniuses?